ViennaAdvantage/Areas/VA012/Scripts/apps/forms/bankstatement.js

```diff
@@ -4227,9 +4453,11 @@
                         _btnIn.attr("v_active", "0");
                     }
                     _txtAmount.getControl().trigger('blur');
+                    //Set the Currency and conversion Type
+                    setCurrencyandConversionType(_scheduleList.toString());
```

Comment:
check schedule list having value in all case or will be null then handle exception

Reply:
Done.